zproject/prod_settings_template.py

```diff
@@ -678,3 +678,8 @@
 
 ## Similarly if you want to set a Privacy Policy.
 # PRIVACY_POLICY = '/etc/zulip/privacy.md'
+
+# Kroki instance URL that is called whenever user wants to render
+# diagrams using textual descriptions
+
+KROKI_RENDERING_URL = "https://kroki.io/{}/svg/{}"
```

Comment:
Can you follow the comment patterns used above? (double-# for anything that's not commented code)

Reply:
This variable should just be `"https://kroki.io/"`. There’s nothing to be gained by exposing the formatting internals, since all Kroki installations will follow the same pattern.